docs/lang/static.md

```diff
@@ -1,16 +1,84 @@
 # Static Timing
 
-By default, Calyx programs use a *latency-insensitive* model of computation.
-This means that the compiler does not track the number of cycles it takes to perform a computation or run a control operator
-In general, latency-insensitivity makes it easier to compose programs together and gives the compiler freedom to schedule operators however it wants.
-However, the generated hardware to schedule the execution may not be efficient–especially if the program can take advantage of the *latency* information.
+By default, Calyx programs use a *latency-insensitive*, or *dynamic*, model of computation.
+This means that the compiler does not know, track, or guarantee the number of cycles it takes to perform a computation or run a control operator.
+This is in contrast to a *latency-sensitive*, or *static*, model of computation, where the number of cycles a component needs is known to, and honored by, the compiler.
 
-More crucially, however, it is impossible for *latency-insensitive* programs to interact with *latency-sensitive* hardware implemented in RTL.
+In general, latency-insensitivity makes it easier to compose programs.
+It grants the compiler freedom to schedule operators however it wants, as long as it meets the program's dataflow constraints.
 
-Calyx uses the `@static` attribute to provide latency information to various constructs and provides strong guarantees about the generated programs.
+However, there are two drawbacks to this approach.
+First, the generated hardware may not be efficient: if the compiler does not know how long computations take, it must schedule them conservatively.
+Second, it is impossible for *latency-insensitive* programs to interact with *latency-sensitive* hardware implemented in RTL;
+this prevents the use of black-box hardware designs.
+
+To address these issues, Calyx provides a `static` keyword that modifies components and groups, along with static variants of other control operators.
```

Comment:
Small nit, but I'd probably use "qualifier" in place of "keyword". It's just a little more specific.